### ISSUE TITLE
Need shortcode for figures

### ISSUE THREAD

**Issue by robdodson** [GoogleChrome/web.dev#815]
We have a fair number of figure/figcaption styles. It would be really nice to shortcode these.

**Comment by robdodson** [comment]
I think there are markdown-it plugins which will do implicit figures. Sam Richard wrote one for his site scaffold project. We should look into using one of those.

**Comment by robdodson** [comment]
We ended up migrating all of our images to `{% Img %}` but a number of them are still wrapped in figures because the either:
- Want a figcaption; OR
- Rely on the figure's CSS to center the image.

So we have a lot of:

```
<figure class="w-screenshot">
  {% Img ... %}
  <figcaption>...</figcaption>
</figure>
```

This feels pretty wonky. We looked into if the `{% Img %}` shortcode should generate a figure/figcaption if someone provides a caption argument. I do think this is ultimately the right approach, but we'll need to unwind the CSS associated with our figure elements so it doesn't require passing in classes that need to live on both the figure and the img. For example, we should make it so the article centers images, as opposed to the figure doing it. That's one less class on figure.
